androidlibrary_lib/src/main/res/layout/progress_indicator_layout.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
```

Comment:
The height is too short when this is used for the initialization fragment, some text is cropped. Please see attached screenshot
![Screenshot_1616481404](https://user-images.githubusercontent.com/8204095/112103948-c650ab80-8b67-11eb-8575-ff0fd1803ccd.png)